vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+
+      clusterHostsActiveStatus:
+        commonlib.panels.generic.stat.info.new(
+          'Active ESXi hosts',
+          targets=[t.clusterHostsActiveCount],
+          description='The number of ESXi hosts that are currently running (responding and not in maintenance mode) within the cluster.'
+        )
+        + stat.options.withGraphMode('none'),
+
+      clusterHostsInactiveStatus:
+        commonlib.panels.generic.stat.info.new(
+          'Inactive ESXi hosts',
+          targets=[t.clusterHostsInactiveCount],
+          description='The number of ESXi hosts that are currently not running (not responding or in maintenance mode) within the cluster.'
+        )
+        + stat.options.withGraphMode('none'),
+
+      clusterResourcePoolsStatus:
+        commonlib.panels.generic.stat.info.new(
+          'Resource pools',
+          targets=[t.clusterResourcePoolsCount],
+          description='The number of resource pools within the cluster (including nested).'
+        )
+        + stat.options.withGraphMode('none'),
+
+      clusterCPULimit:
+        barGauge.new(title='Cluster CPU limit')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPULimit,
+        ])
+        + barGauge.panelOptions.withDescription('The available CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.thresholds.withSteps([
+          barGauge.thresholdStep.withColor('super-light-green'),
+        ])
+        + barGauge.standardOptions.withUnit('rotmhz'),
+
+      clusterCPUEffective:
+        barGauge.new(title='Cluster CPU effective')
+        + barGauge.queryOptions.withTargets([
+          t.clusterCPUEffective,
+        ])
+        + barGauge.panelOptions.withDescription('The effective CPU capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.thresholds.withSteps([
+          barGauge.thresholdStep.withColor('super-light-green'),
+        ])
+        + barGauge.standardOptions.withUnit('rotmhz'),
+
+      clusterCPUUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'Cluster CPU utilization',
+          targets=[t.clusterCPUUtilization],
+          description='The CPU utilization percentage of the cluster.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      clusterMemoryLimit:
+        barGauge.new(title='Cluster memory limit')
+        + barGauge.queryOptions.withTargets([
+          t.clusterMemoryLimit,
+        ])
+        + barGauge.panelOptions.withDescription('The available memory capacity of the cluster.')
+        + barGauge.options.withOrientation('horizontal')
+        + barGauge.standardOptions.thresholds.withSteps([
+          barGauge.thresholdStep.withColor('super-light-green'),
+        ])
+        + barGauge.standardOptions.withUnit('bytes'),
```

Comment:
Minor potential issue on these two panels, with the legend effectively only supporting a single entry before you have to scroll.
They might benefit from having their height increased a bit, or changing the legend to a list style 
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/758ecaa7-9770-4629-a32f-ae3179ceac0f)